content/en/docs/Concepts/olm-architecture/operator-registry/building-a-catalog.md

```diff
@@ -77,6 +77,106 @@ opm index add --bundles quay.io/my-container-registry-namespace/my-manifest-bund
 <pre></pre>
 For more detail on using `opm` to generate index images, take a look at the [documentation](https://github.com/operator-framework/operator-registry/blob/master/docs/design/opm-tooling.md).
 <pre></pre>
+
+# Methods for Specifying Updates
+
+## Replaces
+
+For explicit updates from one CSV to another, you can specify the CSV name to replace in your CSV as
+such:
+
+```yaml
+metadata:
+   name: myoperator.v1.0.1
+spec:
+   replaces: myoperator.v1.0.0
+```
+
+In order for `myoperator.v1.0.1` to be added to the index successfully, `myoperator.v1.0.0` needs to
+have already been added to that index.
+
+An update sequence of bundles created via `replaces` will have updates step through each version in
+the chain. For example, given
+
+```txt
+myoperator.v1.0.0 -> myoperator.v1.0.1 -> myoperator.v1.0.2
+```
+
+A subscription on `myoperator.v1.0.0` will update to `myoperator.v1.0.2` through `myoperator.v1.0.1`.
+
+Installing from the UI today will always install the latest of a given channel. However, installing
+specific versions is possible with this update graph by modifying the `startingCsv` field
+of the subscription to point to the desired CSV name.
+
+## Skips
+
+In order to skip through certain updates you can specify a list of CSV names to be skipped as such:
+
+```yaml
+metadata:
+   name: myoperator.v1.0.3
+spec:
+   replaces: myoperator.v1.0.0
+   skips:
+   - myoperator.v1.0.1
+   - myoperator.v1.0.2
+```
+
+Using the above graph, this will mean subscriptions on `myoperator.v1.0.0` can update directly to
+`myoperator.v1.0.3` without going through `myoperator.v1.0.1` or `myoperator.v1.0.2`. Installs
+that are already on `myoperator.v1.0.1` or `myoperator.v1.0.2` will still be able to update to
+`myoperator.v1.0.3`.
+
+This is particularly useful if `myoperator.v1.0.1` and `myoperator.v1.0.2` are affected by a CVE
+or contain bugs.
+
+Skipped CSVs do not need to be present in an index prior to adding a given version.
+
+## SkipRange
```

Comment:
Should add a note/warning here is that CSV will need to specify `version` in order to use `skipRange`. That field is not required at the moment.